plugins/sentinelone/komand_sentinelone/connection/connection.py

```diff
@@ -163,6 +164,44 @@ def agents_summary(self, site_ids, account_ids):
     def agents_action(self, action: str, agents_filter: str):
         return self._call_api("POST", f"agents/actions/{action}", {"filter": agents_filter})
 
+    def agents_action(self, action: str, agents_filter: str):
+        return self._call_api("POST", f"agents/actions/{action}", {"filter": agents_filter})
+
+    def fetch_file_by_agent_id(self, agent_id: str, file_path: str, password: str):
+        response = self._call_api(
+            "POST", f"agents/{agent_id}/actions/fetch-files", {"data": {"password": password, "files": [file_path]}}
+        )
+        if len(response.get("errors", [])) == 0:
+            return True
+
+        errors = "\n".join(response.get("errors"))
+        raise PluginException(
+            cause="An error occurred when trying to fetch file.",
+            assistance=f"Check the error information and adjust inputs accordingly",
+            data=errors,
+        )
+
+    def run_remote_script(self, user_filter: dict, data: dict) -> dict:
+        endpoint = "remote-scripts/execute"
+        response = self._call_api(
+            "POST",
+            endpoint,
+            {"filter": user_filter, "data": data},
+        )
+        affected = 0
+        if len(response.get("errors", [])) == 0:
```

Comment:
```suggestion
        if not response.get("errors", []):
```

Reply:
@igorski-r7 I tried your suggestion but I've had to revert - the original why I had it avoids the following pylint error getting thrown:
` pylint: use-implicit-booleaness-not-len / Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty (col 11)`
Let me know if you are happy with it left in the original format or happy to hear other suggestions :)